blocks/api/registration.js

```diff
@@ -94,16 +95,7 @@ export function registerBlockType( name, settings ) {
 		...settings,
 	};
 
-	if ( typeof name !== 'string' ) {
-		console.error(
-			'Block names must be strings.'
-		);
-		return;
-	}
-	if ( ! /^[a-z][a-z0-9-]*\/[a-z][a-z0-9-]*$/.test( name ) ) {
-		console.error(
-			'Block names must contain a namespace prefix, include only lowercase alphanumeric characters or dashes, and start with a letter. Example: my-plugin/my-custom-block'
-		);
+	if ( ! validatePluginId( name, 'Block names' ) ) {
```

Comment:
`validatePluginId` doesn't perfectly fit here, maybe `validateNamespacedId` or something along those lines?

Reply:
Thanks for the suggestion, I was also thinking the name wasn't appropriate anymore in this context.